Library/Homebrew/formula.rb

```diff
@@ -1760,6 +1761,10 @@ def generate_completions_from_executable(*commands,
         "--shell=#{shell}"
       elsif shell_parameter_format == :none
         nil
+      elsif shell_parameter_format == :click
```

Comment:
Should add this to the method's `@param shell_parameter_format` comment as well.